RecoEgamma/EgammaTools/src/EgammaRegressionContainer.cc

```diff
@@ -0,0 +1,69 @@
+#include "RecoEgamma/EgammaTools/interface/EgammaRegressionContainer.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "CondFormats/DataRecord/interface/GBRDWrapperRcd.h"
+#include "CondFormats/EgammaObjects/interface/GBRForestD.h"
+
+EgammaRegressionContainer::EgammaRegressionContainer(const edm::ParameterSet& iConfig):
+  outputTransformer_(iConfig.getParameter<double>("rangeMin"),iConfig.getParameter<double>("rangeMax")),
+  forceHighEnergyTrainingIfSaturated_(iConfig.getParameter<bool>("forceHighEnergyTrainingIfSaturated")),
+  lowEtHighEtBoundary_(iConfig.getParameter<double>("lowEtHighEtBoundary")),
+  ebLowEtForestName_(iConfig.getParameter<std::string>("ebLowEtForestName")),
+  ebHighEtForestName_(iConfig.getParameter<std::string>("ebHighEtForestName")),
+  eeLowEtForestName_(iConfig.getParameter<std::string>("eeLowEtForestName")),
+  eeHighEtForestName_(iConfig.getParameter<std::string>("eeHighEtForestName")),
+  ebLowEtForest_(nullptr),ebHighEtForest_(nullptr),
+  eeLowEtForest_(nullptr),eeHighEtForest_(nullptr)
+{
+
+}
+
+edm::ParameterSetDescription EgammaRegressionContainer::makePSetDescription()
+{
+  edm::ParameterSetDescription desc;
+  desc.add<double>("rangeMin",-1.);
+  desc.add<double>("rangeMax",3.0);
+  desc.add<double>("lowEtHighEtBoundary",50.);
+  desc.add<bool>("forceHighEnergyTrainingIfSaturated",false);
+  desc.add<std::string>("ebLowEtForestName","electron_eb_ECALTRK_lowpt");
+  desc.add<std::string>("ebHighEtForestName","electron_eb_ECALTRK_lowpt");
```

Comment:
to avoid future confusion, please update this to ```electron_eb_ECALTRK ```

Reply:
done